docs/notebooks/sdba-advanced.ipynb

```diff
@@ -307,6 +307,101 @@
     "\n",
```

Comment:
<p>Here is another short example.. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>